source/ranges.tex

```diff
@@ -13302,7 +13302,7 @@
     requires @\libconcept{view}@<V> && is_object_v<Pred>
   class chunk_by_view : public view_interface<chunk_by_view<V, Pred>> {
     V @\exposid{base_}@ = V();                       // \expos
-    @\exposid{copyable-box}@<Pred> @\exposid{pred_}@ = Pred();  // \expos
+    @\exposid{movable-box}@<Pred> @\exposid{pred_}@ = Pred();  // \expos
```

Comment:
A space is needed before the comment to keep them aligned.

Reply:
Will fix!